frontend/src/app/shared/components/breadcrumbs/op-breadcrumbs.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="op-breadcrumbs">
+  <!-- Desktop Version -->
+  <nav [attr.aria-label]="text.breadcrumb" class="hidden-for-mobile">
+    <ol class="op-breadcrumb-items">
```

Comment:
Following our usual BEM-like notation, this should be
```suggestion
    <ol class="op-breadcrumbs--items">
```